libkbfs/keybase_daemon_local.go

```diff
@@ -495,3 +495,9 @@ func newKeybaseDaemonLocal(codec kbfscodec.Codec,
 		favoriteStore: favoriteStore,
 	}
 }
+
+// EstablishMountDir asks the service for the current mount path
+// and sets it if not established
+func (k *KeybaseDaemonLocal) EstablishMountDir(ctx context.Context) (string, error) {
```

Comment:
Please group this with the other `KeybaseDaemonLocal` functions (above `Shutdown`).